mobile/src/screens/JoinGame/index.js

```diff
@@ -43,12 +43,12 @@ export default function JoinGame({
 
             case GameSessionState.CHOOSE_TRICKIEST_ANSWER:
             case GameSessionState.PHASE_2_DISCUSS:
-                navigation.push("PhaseTwoBasicGamePlay")
+                navigation.navigate("PhaseTwoBasicGamePlay")
```

Comment:
I highly recommend against this as although phase one and phase two are using the same class instructions, they are two separate screens and should be interacted the same way. If the issue is `selectedAnswerIndex`, then that's something that can be computed for phase 2 based on the data that we have already.

In addition, `navigate` pops back in the stack to that view which messes up the navigation stack and can cause some other issues

Reply:
Good to know, we'll have to find another way. I'm closing this PR without merge. Thanks for clarifying, Mani.